hw/dv/tools/xcelium/common_cov.vRefine

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
+<refinement-file-root>
+  <information comment-version="1" creation-time="Wed 10 Feb 2021 04:11:52 PST" creator="rasmus.madsen" csCheck="false" modified-by="rasmus.madsen" modify-time="Wed 10 Feb 2021 05:18:29 PST" save-ref-method="seq" tool-version="Cadence vManager20.03" rules-signature-c="1ead9ec7f5923387d479a9c435d026b0">
+    <ucm-files>
+      <ucm-file domain="icc" modelCheckSum="1468209578" path="/wdc/proj/cto/top/work/rasmus.madsen/opentitan/golden/hw/ip/aes/dv/scratch/tools_cov/aes-sim-xcelium/cov_merge/merged/icc_578319aa_69124526.ucm"></ucm-file>
+    </ucm-files>
+    <ccf-files>
+      <ccf-file content="//only collect toggle coverage on dut;deselect_coverage -toggle -instance tb.dut.*;// only collect toggle coverage on ports;set_toggle_portsonly;// enable toggle scoring of structs and multidim arrays + MDA structs;set_toggle_scoring -sv_mda -sv_mda_of_struct;// filter glitches;set_toggle_strobe 0ps;" path="/wdc/proj/cto/top/work/rasmus.madsen/opentitan/golden/hw/dv/tools/xcelium/xcelium.ccf"></ccf-file>
```

Comment:
can we break this to multiple lines for readability?
```suggestion
      <ccf-file content="
      //only collect toggle coverage on dut;
      deselect_coverage -toggle -instance tb.dut.*;
      // only collect toggle coverage on ports;
      set_toggle_portsonly;
     ...

Reply:
this file is auto generated

Reply:
Sometimes, it still works when we do some minor modification. at least it works for VCS cov exclusion file
It's good to remove IP specific content and try to run with another block before we add it into common area.